pkgs/development/tools/misc/macdylibbundler/default.nix

```diff
@@ -11,6 +11,9 @@ stdenv.mkDerivation {
     sha256 = "149p3dcnap4hs3nhq5rfvr3m70rrb5hbr5xkj1h0gsfp0d7gvxnj";
   };
 
+  # at run-time, macdylibbundler wants otool to be on the PATH
+  propagatedBuildInputs = [ cctools ];
```

Comment:
Why not use wrapPath? propagatedBuildInputs is more for libs.

Reply:
If that’s idiomatic, then yes, sounds good as well!

I am not sure I can prepare (and test) that patch, though, since I don't have access to a mac machine myself (but I don’t expect anyone else to care enough about this to do it for me, of course.)

Reply:
Just build the package and start it or do I need to do something specific?

Reply:
I guess, yes.

A more complex test is making sure you don’t have `otools` on your path yourself, and then running it some binary ((could it be itself, I guess), i.e.
```
dylibbundler -b -x $(which dylibbundler) -d /tmp/foo
```
and see if it complains about missing `otool` or if it successfully copied stuff to `/tmp/foo`.

Reply:
Works for me but please use makeWrapper.

Reply:
Like this?

Reply:
Like this?

Reply:
looks good to me.